recipes/reduct-cpp/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import (
+    copy,
+    get,
+    rmdir,
+)
+
+required_conan_version = ">=2"
+
+
+class ReductCppConan(ConanFile):
+    name = "reduct-cpp"
+    description = "Reduct Storage Client SDK for C++"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.reduct.store/docs/getting-started/with-cpp"
+    topics = ("reduct-storage", "http-client", "http-api")
+    package_type = "library"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_chrono": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_chrono": False,
+    }
+
+    implements = ["auto_shared_fpic"]
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            # Chrono is always used on Windows
+            del self.options.with_chrono
+
+    @property
+    def _with_chrono(self):
+        # True because removal means that chrono is always used
+        return self.options.get_safe("with_chrono", True)
```

Comment:
Could you provide more info about this? I'm sure I'm missing something, but I didn't find any evidence about this in the upstream code. If that, it could be great to add more context as a comment 🙏